tests/integration.rs

```diff
@@ -91,6 +97,35 @@ mod integration_tests {
             run(SLIGHT, vec!["-c", file_config, "run", "-m", KV_TEST_MODULE]);
             Ok(())
         }
+
+        #[test]
+        #[cfg(unix)] // Redis is not available on Windows: https://redis.io/docs/getting-started/installation/install-redis-on-windows/
+        fn redis_test() -> Result<()> {
+            // make sure redis server is running
+            let port = get_random_port();
```

Comment:
Why use a random port? Does this ensure it is not being used?

Reply:
yep